AnnoyancesFilter/Popups/sections/popups_specific.txt

```diff
@@ -8,6 +8,7 @@
 ! SECTION: Popups - Regular rules
 !
 bitrix24.ru##.helpdesk-notification-block
+lettuceclub.net###rewardModal
```

Comment:
unfortunately, I can't reproduce this popup, could you send a screenshot with devtools open?

Reply:
I can not reproduce now too.
I think the website changed.

Reply:
well that happens, then you could delete these rules and add a rule for subscription